crates/paralegal-policy/src/flows_to.rs

```diff
@@ -13,31 +14,45 @@ pub struct CtrlFlowsTo {
     /// The indexes of [`DataSource`]s in the controller.
     pub sources: Arc<IndexedDomain<DataSource>>,
 
-    /// The indexes of [`DataSink`]s in the controller.
-    pub sinks: Arc<IndexedDomain<DataSink>>,
+    /// The indexes of [`CallSiteOrDataSink`]s in the controller.
+    pub sinks: Arc<IndexedDomain<CallSiteOrDataSink>>,
 
     /// The transitive closure of the [`Ctrl::data_flow`] relation.
+    /// If a source flows to a [`DataSink::Argument`], it also flows into its CallSite.
     ///
     /// See the [`IndexMatrix`] documentation for details on how to
     /// query this representation of the relation.
-    pub flows_to: IndexMatrix<DataSourceIndex, DataSink>,
+    pub data_flows_to: IndexMatrix<DataSourceIndex, CallSiteOrDataSink>,
+
+    /// The transitive closure of the [`Ctrl::data_flow`] and [`Ctrl::ctrl_flow`] relations.
```

Comment:
Nice. I'd add a half-sentence like "representing mixed flows of data and control."

Reply:
Done